Mage.Tests/src/test/java/org/mage/test/cards/single/m21/EnthrallingHoldTest.java

```diff
@@ -0,0 +1,57 @@
+package org.mage.test.cards.single.m21;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class EnthrallingHoldTest extends CardTestPlayerBase {
+
+    @Test
+    public void testTappedTarget_untapped_doesNotFizzle() {
+        addCard(Zone.BATTLEFIELD, playerB, "Traxos, Scourge of Kroog");
+
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 6);
+        addCard(Zone.HAND, playerA, "Enthralling Hold");
```

Comment:
Add ability text/cost as comments (only those abilities that are used in the test). Example:

![shot_200629_122224](https://user-images.githubusercontent.com/8344157/85990648-63c8e580-ba03-11ea-9700-85587071b049.png)

Reply:
fixed